src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Generated/IOperationsOperations.cs

```diff
@@ -0,0 +1,47 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
@avneeshrai missing license header.
Chances are your swagger does not have the right License Header Constant, please double check, I believe the constant that you need to use in your swagger is "MIT-no-version"